src/store/modules/accounts/hdWallet.js

```diff
@@ -6,17 +6,16 @@ import {
   buildTx,
 } from '@aeternity/aepp-sdk';
 
-import { useModals, useAeSdk } from '@/composables';
+import { useModals } from '@/composables';
 import {
   ACCOUNT_HD_WALLET,
   MODAL_CONFIRM_RAW_SIGN,
   MODAL_CONFIRM_TRANSACTION_SIGN,
   PROTOCOL_AETERNITY,
+  PROTOCOL_BITCOIN,
 } from '@/constants';
-import {
-  getHdWalletAccount,
-} from '@/popup/utils';
 import { isTxOfASupportedType } from '@/protocols/aeternity/helpers';
+import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
```

Comment:
I suggest to have this unwritten rule that first we import the common things, like types, utils, libs and then the protocol specific. WHat do you think?

Reply:
All these imports (types, utils, and libs) are generally the most edited imports. keeping them first would lead to conflicts between parallel commits. I suggest keeping them at last except the types import that could be at the first.
Thus we can minimize the merge conflicts.